client/src/components/AuthModal.js

```diff
@@ -24,14 +26,20 @@ const AuthModal = ({setShowModal, isSignUp}) => {
                 setError('Passwords need to match!');
                 return;
             }
-            console.log('make a post request to our database');
-            const response = await axios.post("http://localhost:8000/signup", {email, password});
+            console.log('make a post request to our database', email, password);
```

Comment:
Hi Inna! I am assuming these console.log has a specific purpose for the backend or can it be removed? thank you